models/issue_reaction.go

```diff
@@ -18,12 +18,12 @@ import (
 // Reaction represents a reactions on issues and comments.
 type Reaction struct {
 	ID               int64              `xorm:"pk autoincr"`
-	Type             string             `xorm:"INDEX UNIQUE(s) NOT NULL"`
+	Type             string             `xorm:"CHAR(32) INDEX UNIQUE(s) NOT NULL"`
```

Comment:
This will fill the column with 32 characters ALWAYS, resulting in something like `test<empty spaces here>`

Reply:
The same issue is already present in master due to your other PR that changed webhook type, namely on Postgres the type is not `gitea` but `gitea<empty spaces>`

Reply:
https://stackoverflow.com/a/4849030/7631980

Reply:
OKay. I will change this to `VARCHAR(32)`